### PR TITLE
Mailman Cargo Unification (Second try)

### PR DESCRIPTION
**Pull request by heresthemailitneverfails**
# About The Pull Request
Makes the Mailman job independent from its Gimmick role

## Why It's Good For The Game
So you can play Mailman for a round-start
Easier to get antag as Mailman

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.
FNV_UnificationMonument

</details>

## Changelog

:cl:
tweak: makes mailman a normal role
add: added bullterrier as mailman "Mail goodie"
/:cl:

### CODE REVIEW

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -39,19 +39,19 @@
...
+#define CHAPLAIN		(1<<11)
+#define CLOWN			(1<<12)
+#define MIME			(1<<13)
+#define ASSISTANT		(1<<14)
+#define GIMMICK 		(1<<15)
+#define BARBER		    (1<<16)
+#define MAGICIAN        (1<<17)
+#define HOBO            (1<<18)
+#define SHRINK          (1<<19)
+#define CELEBRITY       (1<<20)
```
Comment (lmenvs) [inline code/__DEFINES/jobs.dm:54]: #define JOB_DISPLAY_ORDER is what you need to add mailman to, not this one

**Review by lmenvs — CHANGES_REQUESTED** [review]

File: code/__DEFINES/jobs.dm (hunk 1 above)
Comment (heresthemailitneverfails) [reply code/__DEFINES/jobs.dm:54]: okay, where is that?

Comment (lmenvs) [reply code/__DEFINES/jobs.dm:54]: revert changes on that, scroll down until you find #define JOB_DISPLAY_ORDER

Comment (heresthemailitneverfails) [reply code/__DEFINES/jobs.dm]: does that look good?

Comment (heresthemailitneverfails) [reply code/__DEFINES/jobs.dm]: frick i accidentally deleted the whole file

Comment (heresthemailitneverfails) [reply code/__DEFINES/jobs.dm]: oh god i'd have to remake the whole file, is it okay if i try again with a new PR and branch??

**Pull request closed without merging**